docs/store-operations/pricing/calculations.mdx

```diff
@@ -8,14 +8,14 @@ The table below lists each price type available on a product. The table is read
 | Product sale price | A product option that overrides the default price |  
 | Variant price | A product option that overrides the product sale price |
 | Variant sale price | A product option that overrides the variant price |
-| Customer group discount | Available in fixed ($5), relative (-$2), or percentage (-25%) amount that overrides the variant sale price. Might apply to one product, category, subcategory, or the entire store's products. |
-| Product bulk pricing | Available in fixed ($5), relative (-$2), or percentage (-25%) amount that overrides the customer group discount. Dependent on the total quantity of products, including SKUs added to the cart. |
 | Price list variants | A price list record requirement that overrides all previous pricing and excludes SKUs from the total number of items for product bulk pricing. 
 | Price list variant sale price | A product option that overrides price list pricing if variant pricing is set and selected. |
 | Price list variant bulk pricing | Available in fixed ($5), relative (-$2), or percentage (-25%) amount that overrides price list sale price on variants. It is dependent on the quantity added to a cart. | 
 | Price list variant product pick list | The product pick list overrides price list bulk pricing and allows you to change the price when a pick list item is selected. | 
 | Product option set (**Deprecated**)| A set of product option facets that may alter the variant price. (This feature is deprecated.) |
 | Product modifier | Available in fixed ($5) or percentage (%10) amount that is added or removed from the total product price and overrides the price list variant product pick list. A modifier includes choices such as adding $5 for insurance. |
+| Customer group discount | Available in fixed ($5), relative (-$2), or percentage (-25%) amount that overrides the variant sale price. Might apply to one product, category, subcategory, or the entire store's products. |
+| Product bulk pricing | Available in fixed ($5), relative (-$2), or percentage (-25%) amount that overrides the customer group discount. Dependent on the total quantity of products, including SKUs added to the cart. |
```

Comment:
```suggestion
| Customer group discount | Available in fixed ($5), relative (-$2), or percentage (-25%) amount that overrides the variant sale price. Might apply to one product, category, subcategory, or the entire store's products. Customer group discounts do not apply to price list prices. |
| Product bulk pricing | Available in fixed ($5), relative (-$2), or percentage (-25%) amount that overrides the customer group discount. Dependent on the total quantity of a single product added to the cart. Product bulk pricing will not apply for SKUs that have price list prices. |
```
@bc-traciporter  what do you think of something like this?  Want to make clear that these aren't in play when there's a price record available.

Reply:
@megdesko that's fine. So we can keep the one table and add this text. I think it works.